draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -4241,15 +4241,15 @@ RESPONSE:
       <t>[RFC Editor: Please replace RFCthis with the published RFC
         number for this document.]</t>
 
-         <section anchor="IANA.CDNI.payload.ci-trigger-command.trigger.v2" title="CDNI ci-trigger-command.trigger.v2 Payload Type">
+         <section anchor="IANA.CDNI.payload.ci-trigger-command.create.v2" title="CDNI ci-trigger-command.create.v2 Payload Type">
```

Comment:
Does "CDNI Payload Type Parameter Registrations" (aboce) need to be updated with create.v2 and modify.v2?

Reply:
updated and fixed ci-trigger-command.* usage across all of the text